lint-go.yml

```diff
@@ -1,6 +1,6 @@
 lint:go:
   stage: code_quality
-  image: "${APP_LINT_IMAGE}"
+  image: "${CI_REGISTRY_IMAGE}/${APP_LINT_IMAGE_NAME}:${CI_COMMIT_SHORT_SHA}"
```

Comment:
I like how it is now (more clear), but I am confused by the commit message that says "shorten" names while this seems longer, what was your intention? to clarify?

only a question about intention, I like it this way

Reply:
You're right, the commit message is a little bit confusing.
The intention is to no longer pass things (variables) that could be accessed from the template as well, in order to be more clear.
Let me change the commit message.